diff --git a/node_modules/@miaz/capacitor-websocket/ios/Plugin/CapacitorWebsocketPlugin.swift b/node_modules/@miaz/capacitor-websocket/ios/Plugin/CapacitorWebsocketPlugin.swift
index 5c86a48..9ff6f79 100644
--- a/node_modules/@miaz/capacitor-websocket/ios/Plugin/CapacitorWebsocketPlugin.swift
+++ b/node_modules/@miaz/capacitor-websocket/ios/Plugin/CapacitorWebsocketPlugin.swift
@@ -1,6 +1,5 @@
 import Foundation
 import Capacitor
-import Network
 import Starscream
 @objc(CapacitorWebsocketPlugin)
 public class CapacitorWebsocketPlugin: CAPPlugin {
diff --git a/node_modules/@miaz/capacitor-websocket/ios/Plugin/WebsocketConnection.swift b/node_modules/@miaz/capacitor-websocket/ios/Plugin/WebsocketConnection.swift
index 1a93cb3..6176d7c 100644
--- a/node_modules/@miaz/capacitor-websocket/ios/Plugin/WebsocketConnection.swift
+++ b/node_modules/@miaz/capacitor-websocket/ios/Plugin/WebsocketConnection.swift
@@ -6,20 +6,19 @@
 //
 
 import Foundation
-import Network
 import Capacitor
 import Starscream
 
 class SocketConnection {
     private var connected: Bool = false
-    var socket: WebSocket?
+    var socket: Starscream.WebSocket?
     var pluginContext: CAPPlugin
     var name: String
 
     init(name: String, plugin: CAPPlugin, request: URLRequest) {
         self.name = name
         self.pluginContext = plugin
-        self.socket = WebSocket(request: request)
+        self.socket = Starscream.WebSocket(request: request)
     }
 
     public func connect() {
@@ -35,33 +34,33 @@ class SocketConnection {
     }
 
     public func bind() {
-        socket?.onEvent = { event in
+        socket?.onEvent = { (event: Starscream.WebSocketEvent) in
             switch event {
-                case .connected(_):
-                    self.connected = true
-                    self.pluginContext.notifyListeners("\(self.name):connected", data: ["iosws": "connected"])
-                case .disconnected(let reason, let code):
-                    self.connected = false
-                    self.pluginContext.notifyListeners("\(self.name):disconnected", data: ["code": code, "reason": reason ])
-                case .text(let string):
-                    self.pluginContext.notifyListeners("\(self.name):message", data: ["data": string])
-                case .binary(let data):
-                    self.pluginContext.notifyListeners("\(self.name):message", data: ["data": data])
-                case .ping(_):
-                    break
-                case .pong(_):
-                    break
-                case .viabilityChanged(_):
-                    break
-                case .reconnectSuggested(_):
-                    break
-                case .cancelled:
-                    self.connected = false
-                case .error(let error):
-                    self.connected = false
-                    self.pluginContext.notifyListeners("\(self.name):error", data: ["cause": error ?? "Unknown Error"])
-                case .peerClosed:
-                    self.pluginContext.notifyListeners("\(self.name):message", data: ["data": "Peer closed"])
+            case .connected(_):
+                self.connected = true
+                self.pluginContext.notifyListeners("\(self.name):connected", data: ["iosws": "connected"])
+            case .disconnected(let reason, let code):
+                self.connected = false
+                self.pluginContext.notifyListeners("\(self.name):disconnected", data: ["code": code, "reason": reason])
+            case .text(let string):
+                self.pluginContext.notifyListeners("\(self.name):message", data: ["data": string])
+            case .binary(let data):
+                self.pluginContext.notifyListeners("\(self.name):message", data: ["data": data])
+            case .ping(_):
+                break
+            case .pong(_):
+                break
+            case .viabilityChanged(_):
+                break
+            case .reconnectSuggested(_):
+                break
+            case .cancelled:
+                self.connected = false
+            case .error(let error):
+                self.connected = false
+                self.pluginContext.notifyListeners("\(self.name):error", data: ["cause": error ?? "Unknown Error"])
+            case .peerClosed:
+                self.pluginContext.notifyListeners("\(self.name):message", data: ["data": "Peer closed"])
             }
         }
     }
